test/bundle.js

```diff
@@ -4,6 +4,4 @@ System.registerDynamic("test/data/test.css!css.js", [], false, function ($__requ
   (function ($__global) {})(this);
 
   return _retrieveGlobal();
-});
-(function(c){if (typeof document == 'undefined') return; var d=document,a='appendChild',i='styleSheet',s=d.createElement('style');s.type='text/css';d.getElementsByTagName('head')[0][a](s);s[a](d.createTextNode(c));})
-("@import \"./dep.css\";body{background-color:red;background-image:url(test/data/x.png)}\n/*# sourceMappingURL=__.css.map */");
```

Comment:
Hrm… I am monkey patching CSSPluginBase in my environment and it builds just fine. I wonder what the difference is?